shell/components/CodeMirror.vue

```diff
@@ -111,13 +127,52 @@ export default {
     }
   },
 
+  async mounted() {
+    document.addEventListener('keyup', this.handleKeyPress);
+    document.addEventListener('focusin', this.focusChanged);
```

Comment:
I strongly discourage adding event listeners to the document. There is a lot of conflict that can arise with event listeners like this, as we found in #13375

Reply:
removed. fixed